ramani-maplibre/src/main/java/org/ramani/compose/MapLibre.kt

```diff
@@ -141,12 +145,13 @@ fun MapLibre(
             maplibreMap.applyUiSettings(currentUiSettings)
             maplibreMap.applyProperties(currentMapProperties)
             maplibreMap.setupLocation(
-                context,
-                style,
-                currentLocationRequestProperties,
-                currentLocationStyling,
-                userLocation,
-                renderMode,
+                context = context,
+                style = style,
+                locationRequestProperties = currentLocationRequestProperties,
+                locationEngine = currentLocationEngine,
+                locationStyling = currentLocationStyling,
+                userLocation = userLocation,
+                renderMode = renderMode
```

Comment:
Nitpick: I like having a comma even for the last element, because it means that adding another element after won't change this line in the diff.

```suggestion
                renderMode = renderMode,
```

Reply:
Sure, I usually do it as well.